packages/core/tangleold/cmanaweightprovider.go

```diff
@@ -238,15 +248,15 @@ func (c *CManaWeightProvider) activityBoundaries() (lowerBoundEpoch, upperBoundE
 }
 
 // clean removes all activity logs for epochs lower than provided bound.
-func (c *CManaWeightProvider) clean(lowerBound epoch.Index) {
+func (c *CManaWeightProvider) clean(cutoffEpoch epoch.Index) {
 	for ei := range c.activityLog {
-		if ei < lowerBound {
+		if ei < cutoffEpoch {
 			delete(c.activityLog, ei)
```

Comment:
c.activityLog should be turned into a ShrinkingMap so we don't leak memory in the map.